auth/forgot-password/ForgotPasswordNavigation.tsx

```diff
@@ -0,0 +1,150 @@
+import { StackNavigatorType } from "@components/Navigation"
+import {} from "@lib/users"
+import { StackScreenProps } from "@react-navigation/stack"
+import { Alert } from "react-native"
+import { EmailAddress, Password, USPhoneNumber } from ".."
+import {
+  AuthVerificationCodeFormView,
+  useAuthVerificationCodeForm
+} from "../VerifyCode"
+import { ForgotPasswordEnvironment } from "./Environment"
+import {
+  ForgotPasswordFormView,
+  useForgotPasswordForm
+} from "./ForgotPasswordForm"
+import {
+  ResetPasswordFormView,
+  useResetPasswordForm
+} from "./ResetPasswordForm"
+
+export type ForgotPasswordParamsList = {
+  forgotPassword: undefined
+  verifyCode: {
+    emailOrPhoneNumber: EmailAddress | USPhoneNumber
+    code?: string
+    newPass?: Password
+  }
+  resetPassword: {
+    initialPassword?: Password
+    emailOrPhoneNumber: EmailAddress | USPhoneNumber
+    code: string
+  }
+}
+
+type ForgotPasswordScreenProps = StackScreenProps<
+  ForgotPasswordParamsList,
+  "forgotPassword"
+> &
+  Pick<ForgotPasswordEnvironment, "initiateForgotPassword">
+
+type VerifyCodeScreenProps = StackScreenProps<
+  ForgotPasswordParamsList,
+  "verifyCode"
+> &
+  Pick<ForgotPasswordEnvironment, "initiateForgotPassword">
+
+type ResetPasswordScreenProps = StackScreenProps<
+  ForgotPasswordParamsList,
+  "resetPassword"
+> &
+  Pick<ForgotPasswordEnvironment, "submitResettedPassword">
+
+export const createForgotPasswordScreens = <T extends ForgotPasswordParamsList>(
+  stack: StackNavigatorType<T>,
+  env: ForgotPasswordEnvironment
+) => {
+  return (
+    <>
+      <stack.Screen name="forgotPassword">
+        {(props: any) => (
+          <ForgotPasswordScreen
+            {...props}
+            initiateForgotPassword={env.initiateForgotPassword}
+          />
+        )}
+      </stack.Screen>
+
+      <stack.Screen name="verifyCode">
+        {(props: any) => <VerifyCodeScreen {...props} />}
+      </stack.Screen>
+
+      <stack.Screen name="resetPassword">
+        {(props: any) => (
+          <ResetPasswordScreen
+            {...props}
+            submitResettedPassword={env.submitResettedPassword}
+          />
+        )}
+      </stack.Screen>
+    </>
```

Comment:
Hide the titles + add options
ForgotPass + Verify: X to close
Reset: Chevron to go back